File: DoneIt_Version_Movil/app/src/main/java/com/herreroco/doneit_version_movil/model/Tarea.kt
package com.herreroco.doneit_version_movil.model

data class Tarea(
    val id_Tarea: Int,
    val titulo: String,
    val descripcion: String?,
    val fecha_Inicio: String,
    val fecha_Fin: String,
    val estado: String,
    val prioridad: String
)

File: DoneItAPI/DoneItAPI/Models/Tarea.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DoneItAPI.Models
{
    [Table("Tareas")]
    public class Tarea
    {
        [Key]
        [Column("id_tarea")]
        public int Id_Tarea { get; set; }

        [Column("titulo")]
        public string Titulo { get; set; }

        [Column("descripcion")]
        public string? Descripcion { get; set; }

        [Column("fecha_inicio")]
        public DateTime? Fecha_Inicio { get; set; }

        [Column("fecha_fin")]
        public DateTime? Fecha_Fin { get; set; }

        [Column("estado")]
        public string Estado { get; set; }

        [Column("prioridad")]
        public string Prioridad { get; set; }

        [Column("id_proyecto")]
        public int Id_Proyecto { get; set; }

       // public Proyecto? Proyecto { get; set; }
    }
}
